kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -4,10 +4,10 @@
 
     <ui-toolbar type="colored" textColor="white">
       <template slot="icon">
-        <ui-icon class="app-bar-icon"><logo/></ui-icon>
+        <img src="../img/instant-school-logo.png" class="app-bar-icon"/>
```

Comment:
The icon needs some styling.

Reply:
![localhost_8000_user_ iphone 5](https://user-images.githubusercontent.com/7193975/32235605-2af7b2fc-be1d-11e7-9cf5-bbac7dbd82d2.png)

Reply:
Used to have styling moved out to a separate branch intentionally